packages/backend/src/routers/projectRouter.ts

```diff
@@ -155,11 +156,23 @@ projectRouter.post(
     unauthorisedInDemo,
     async (req, res, next) => {
         try {
-            // Runs async - error will appear on status endpoint
+            const jobUuid = uuidv4();
+
+            projectService.startJob(jobUuid, req.params.projectUuid); // So we don't get a 404 when requesting this jobUuid
```

Comment:
I would have expected for the startJob to generate the job uuid and return it. Is there a reason why is passed as an arg ?

Reply:
There is not a good reason, initially that startJob didn't exist, and it was added to avoid a race condition, but what you suggested makes more sense. I'll change it